libclc/utils/libclc-remangler/LibclcRemangler.cpp

```diff
@@ -924,11 +902,9 @@ class LibCLCRemangler : public ASTConsumer {
       FuncList.push_back(&Func);
 
     bool Success = true;
-    unsigned NumProcessed = 0;
-    for (auto *Func : FuncList) {
-      Success = remangleFunction(*Func, M) && Success;
-      ++NumProcessed;
-    }
+    size_t N = FuncList.size();
```

Comment:
You should remove this variable.
1. It was declared here because it was used in the loop. The declaration should be moved closer to use.
2. Only one use is left, so there is no need to declare a variable.

Reply:
:+1: